packages/components/tests/integration/components/hds/table/index-test.js

```diff
@@ -35,7 +35,12 @@ const setSortableTableData = (context) => {
   ]);
   context.set('columns', [
     { key: 'artist', label: 'Artist', isSortable: true },
-    { key: 'album', label: 'Album', isSortable: true },
+    {
+      key: 'album',
+      label: 'Album',
+      isSortable: true,
+      sortingFunction: 'mySortingFunction',
```

Comment:
@MelSumner this will not work, because you're passing a string, not a function (try to change it to `'foo'` and you will see tests pass anyway)